frontend/views/containers/modals/JoinGroup.vue

```diff
@@ -0,0 +1,223 @@
+<template lang='pug'>
+modal-base-template
+  i18n(tag='h1') How to join a group
+
+  .wrapper
+    .slider
+      .slide(v-observer:0='updateIndicator' :id='config[0]')
+        .slide-img
+          svg-invitation
+
+        i18n(tag='h4') 1. Get an invitation
+        i18n(tag='p') For now, the only way to join a group is to get an invitation. Don’t know anyone using Group Income?
+        i18n(tag='button' class='link' @click='showCreateModal') Create your own group
+
+      .slide(v-observer:1='updateIndicator' :id='config[1]')
+        .slide-img
+          svg-proposal
+
+        i18n(tag='h4') 2. Wait for the group vote
+        i18n(tag='p') On Group Income, every major decision goes through a voting process. This includes adding new members.
+
+      .slide(v-observer:2='updateIndicator' :id='config[2]')
+        .slide-img
+          svg-access
+
+        i18n(tag='h4') 3. Use your unique access link
+        i18n(tag='p') Once the group agrees that you should join them, a unique access link will be generated, giving you instant access to the group
+
+    .dots
+      a.dot(
+        v-for='(link, i) in config'
+        :class='{ "is-active": ephemeral.indicator == i }'
+        :href='"#"+link'
+        :aria-label='L("Go to {num} slider", { num: i + 1 })'
+      )
+</template>
+
+<script>
+import ModalBaseTemplate from '@components/Modal/ModalBaseTemplate.vue'
+import { REPLACE_MODAL } from '@utils/events.js'
+import sbp from '~/shared/sbp.js'
+import SvgAccess from '@svgs/access.svg'
+import SvgInvitation from '@svgs/invitation.svg'
+import SvgProposal from '@svgs/proposal.svg'
+
+export default {
+  name: 'JoinGroupModal',
+  components: {
+    ModalBaseTemplate,
+    SvgAccess,
+    SvgInvitation,
+    SvgProposal
+  },
+  data () {
+    return {
+      ephemeral: {
+        indicator: 0
+      },
+      config: [
+        'get-an-invitation',
+        'wait-for-you-group-vote',
+        'use-your-unique-access-link'
+      ]
+    }
+  },
+  // TODO move this into slider component if we have more than one
+  directives: {
+    observer: {
+      inserted: (el, { value, arg }) => {
+        try {
+          const io = new window.IntersectionObserver(
+            (elements) => {
+              if (elements[0].intersectionRatio >= 0.5) { // Wait for the middle of the screen
+                value(parseInt(arg))
+              }
+            },
+            { threshold: [0.5] }
+          )
+
+          io.observe(el)
+
+          el.stop = () => {
+            io.disconnect()
+          }
+        } catch (error) {
+          console.error(error.message, '\n polyfill: https://github.com/w3c/IntersectionObserver')
+        }
+      }
+    },
+    unbind (el) {
+      el.stop()
+    }
```

Comment:
Two issues:

1. `unbind` needs to be defined next to `inserted`, inside the `observer` object/directive
2. Defining a property on an [`Element`](https://developer.mozilla.org/en-US/docs/Web/API/Element) is risky, since there can be a name conflict if the standards people ever add "stop" as a property. So can you rename it to something like `.$stop` or `.giUnbind` to avoid potential name conflicts?